change_notes/2024-06-03-constexpr-variable.md

```diff
@@ -0,0 +1,3 @@
+ - `A7-1-2` - `VariableMissingConstexpr.ql`:
+   - Remove false positives for compiler generated variables
```

Comment:
please add the issue # as well here, bc that renders in the release notes nicely